README.md

```diff
@@ -32,6 +34,21 @@ php artisan backpack:add-sidebar-content "<li class='nav-item'><a class='nav-lin
 php artisan db:seed --class="Backpack\Settings\database\seeds\SettingsTableSeeder"
 ```
 
+### [OPTIONAL] Configuration
```

Comment:
I think it would be easier to follow if we include this step as part of the installation instructions, so that those who do need to change the table name don't get to publish everything, run the migration, then have to re-do everything all over again. So maybe something like this instead?

```bash
``` bash
# install the package
composer require backpack/settings

# [optional] change the table name if "settings" is not ok with you
php artisan vendor:publish --provider="Backpack\Settings\SettingsServiceProvider" --tag="config"
# then change the table name in `config/backpack/settings.php`

# publish and run the migration
php artisan vendor:publish --provider="Backpack\Settings\SettingsServiceProvider"
php artisan migrate

# [optional] add a menu item for it to the sidebar_content file
php artisan backpack:add-sidebar-content "<li class='nav-item'><a class='nav-link' href='{{ backpack_url('setting') }}'><i class='nav-icon la la-cog'></i> <span>Settings</span></a></li>"

# [optional] insert some example dummy data to the database
php artisan db:seed --class="Backpack\Settings\database\seeds\SettingsTableSeeder"
```

Reply:
Feels like step-by-step we're moving towards a difficult setup, that will need a `php artisan backpack:settings:install` 😅

Reply:
I agree with your changes in the Readme, but I think a install command for this is probably far, or not priority. I don't see many people using this configuration option. But I do see people using spatie settings and backpack settings, because they "complete" each other, one is «entity/model settings» other is a «general setting panel». 

Remember where you saw both of them together ? 🙃 Hope I take some time to finish up that package, I think would make this obsolete and spatie for people working with backpack exclusively. 

For now, I've just addressed your concerns in this PR 👍